plugins/doors/sv_plugin.lua

```diff
@@ -172,7 +157,13 @@ function PLUGIN:CanPlayerAccessDoor(client, door, access)
 	local faction = door:getNetVar("faction")
 
 	-- If the door has a faction set which the client is a member of, allow access.
-	if (faction and client:Team() == faction) then
+	if(istable(faction)) then
+		for k, v in pairs(faction) do
+			if(client:Team() == v) then
+				return true
+			end
+		end
+	elseif (faction and client:Team() == faction) then
 		return true
 	end
```

Comment:
You should consider accessing the factions by index to prevent having to iterate over them.